code/modules/reagents/reagent_containers/food/snacks.dm

```diff
@@ -65,29 +65,29 @@
 
 	if(iscarbon(M))
 		var/mob/living/carbon/C = M
-		var/fullness = C.nutrition + (C.reagents.get_reagent_amount(/datum/reagent/consumable/nutriment) * 25)
+		var/fullness = C.nutrition + (C.reagents.get_reagent_amount(/datum/reagent/consumable/nutriment) * 25) + (reagents.get_reagent_amount(/datum/reagent/consumable/nutriment) * 25)
```

Comment:
What's this formula coming from? It looks like nutriment is 7.5 nutrition / 0.4u, which is significantly less than 50/1. 25/1 is wrong too but it's closer and still overshoots.

Reply:
Oh oops I just glazed entirely over the two reagent holders being separate. Make sure you're only accounting for the nutriment you'll actually get from a bite instead of the entire container.